tests/test_featureprofiletools_value_renamer.py

```diff
@@ -17,14 +19,19 @@
 DIR_WITH_OUTPUT_GOLD_STANDARD_INVENTORIES = DIR_WITH_OUTPUT_GOLD_STANDARD_FILES / "inventories"
 
 
-def test_rename_value_in_profiles_and_inventories():
-    # First change files in tmp up to their default variants
+@pytest.fixture
+def create_value_renamer():
     value_renamer = ValueRenamer(
         input_feature_profiles_dir=DIR_WITH_INPUT_FEATURE_PROFILES,
         output_feature_profiles_dir=DIR_WITH_TEST_UPDATE_PROFILES_INVENTORIES,
         input_inventories_dir=DIR_WITH_INPUT_INVENTORIES,
         output_inventories_dir=DIR_WITH_TEST_UPDATE_PROFILES_INVENTORIES / "inventories",
     )
+    return value_renamer
+
+
+def test_rename_value_in_profiles_and_inventories(create_value_renamer):
+    value_renamer = create_value_renamer
     value_renamer.rename_value_in_profiles_and_inventories(
```

Comment:
```suggestion
def test_rename_value_in_profiles_and_inventories(value_renamer):
    value_renamer.rename_value_in_profiles_and_inventories(
```
As far as I remember, you don't need any assignment. Just use the fixture.

Reply:
Fixture assignments are now removed in all the tests